cfgov/jinja2/v1/_includes/macros/util/format/datetime.html

```diff
@@ -64,7 +64,7 @@
 
    Format a date for datetime metadata in the format of:
 
-       YYYY-MM-DDTHH:MM:SS.MMMM
+       YYYY-MM-DDTHH:MM:SS
```

Comment:
Could update this to show the time zone offset that appears at the end, too.

Reply:
+1 to including the time zone.

Reply:
Done! As `-Z`